sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -335,7 +335,7 @@ than only sub-groups.
 ----
 id_type get_group_id() const;
 ----
-_Returns_: An `id` representing the index of the ballot-group.
+_Returns_: An `id` representing the index of the ballot-group in a parent group.
```

Comment:
Note for reviewers: I'm not entirely sure if this clarification is needed, perhaps that piece of information is implied and should be obvious, but I though that it would be better to state it explicitly. 

If it is decided that it makes sense to add such clarification, then I should have another pass over the doc, because I haven't been consistent with adding it.

Reply:
I am not even sure the clarification is that clear. What does it mean to have an "index in a parent group". You can create multiple ballot-groups from the same sub-group, and they may have the same indices.

Reply:
The clearest thing to do here might be to say something really precise (and verbose), like:

```suggestion
_Returns_: An `id` representing the index of the ballot-group to which the
calling work-item belongs, as determined by the work-item's associated
`predicate` value.
```

...but I appreciate this isn't perfect either.  One of the things we've been exploring is to replace the concept of a `ballot-group` with "the result of a logical partitioning", which might make this easier to explain.  This would then read as "the index of the logical partition to which..."